src/components/data-table/column-sorting/column-sort.service.ts

```diff
@@ -0,0 +1,47 @@
+import {Injectable} from '@angular/core';
+import {Subject} from 'rxjs/Subject';
+import {
+  SortDirection,
+  IColumnSortingModel
+} from './column-sort';
+
+/**
+ * Service handling column sort change events as an observable stream.
+ */
+@Injectable()
+export class MdDataColumnSortingService {
+  public sortingColumn$: Subject<IColumnSortingModel>
```

Comment:
Use the more concise typescript class member initializer and remove constructor, e.g.

```typescript
public sortingColumn$: Subject<IColumnSortingModel> = new Subject();
```